doc/pip_and_pep_668.rst

```diff
@@ -0,0 +1,72 @@
+Pip installation after PEP 668
+==============================
+
+`PEP-668`_ introduced `externally managed environments <externally-managed-environments>`_ to Python packaging.
+
+``rosdep`` is designed to use pip as an alternative system package manager, rosdep installation of pip packages requires installing packages globally as root.
+Starting with Python 3.11, `PEP-668`_ compliance requires you to allow pip to install alongside externally managed packages using the ``break-system-packages`` option.
+
+There are multiple ways to configure pip so that rosdep will succeed.
+
+
+Configure using environment variable
+------------------------------------
+
+This is the way that we recommend configuring pip for rosdep usage.
+We recommend configuring pip using the system environment.
+Setting environment variables in your login profile, ``PIP_BREAK_SYSTEM_PACKAGES`` in your environment.
+The value of the environment variable can be any of ``1``, ``yes``, or ``true``.
+The string values are not case sensitive.
+
+``rosdep`` is designed to use ``sudo`` in order to gain root privileges for installation when not run as root.
+If your system's sudo configuration prohibits the passing of environment variables
```

Comment:
it looks like this sentence was cut off.

Reply:
lol good catch. I'm sure that I went to look up how to reference a section in the doc below and got distracted because that's exactly what I just did again.